subs2srs.lua

```diff
@@ -48,8 +48,9 @@ local secondary_sid = require('subtitles.secondary_sid')
 local platform = require('platform.init')
 local forvo = require('utils.forvo')
 local subs_observer = require('subtitles.observer')
-local menu
-
+local menu, quick_menu, quick_menu_card
+local n_lines
+local n_cards = 1
```

Comment:
These two values are logically tied together, so it would be appropriate to encapsulate them in one object.